internal/persistence/persistors/mysqlstore/store_organization.go

```diff
@@ -0,0 +1,290 @@
+		}
+
+		if filters.CreatedBy.Valid {
+			queryMods = append(queryMods, mysqlmodel.OrganizationWhere.CreatedBy.EQ(filters.CreatedBy))
+		}
+
+		if filters.LastUpdatedBy.Valid {
+			queryMods = append(queryMods, mysqlmodel.OrganizationWhere.LastUpdatedBy.EQ(filters.LastUpdatedBy))
+		}
+	}
+
+	q := mysqlmodel.Organizations(queryMods...)
+	totalCount, err := q.Count(ctx, ctxExec)
+	if err != nil {
+		return nil, fmt.Errorf("get organizations count: %v", err)
+	}
+
+	page := pagination.Page
+	maxRows := pagination.MaxRows
+	if filters != nil {
+		if filters.Page.Valid {
+			page = filters.Page.Int
+		}
+		if filters.MaxRows.Valid {
+			maxRows = filters.MaxRows.Int
+		}
+	}
+
+	pagination.SetQueryBoundaries(page, maxRows, int(totalCount))
+
+	queryMods = append(queryMods, qm.Limit(pagination.MaxRows), qm.Offset(pagination.Offset))
+	q = mysqlmodel.Organizations(queryMods...)
+
+	if err = q.Bind(ctx, ctxExec, &res); err != nil {
+		return nil, fmt.Errorf("get organizations: %v", err)
+	}
+
+	pagination.RowCount = len(res)
+	paginated.Organizations = res
+	paginated.Pagination = pagination
+
+	return &paginated, nil
+}
+
+func (m *Repository) CreateOrganization(ctx context.Context, tx persistence.TransactionHandler, organization *model.Organization) (*model.Organization, error) {
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return nil, fmt.Errorf("extract context executor: %v", err)
+	}
+
+	entry := mysqlmodel.Organization{
+		Name: organization.Name,
+	}
+	if err = entry.Insert(ctx, ctxExec, boil.Infer()); err != nil {
+		return nil, fmt.Errorf("insert organization: %v", err)
+	}
+
+	organization, err = m.GetOrganizationById(ctx, tx, entry.ID)
+	if err != nil {
+		return nil, fmt.Errorf("get organization by id: %v", err)
+	}
+
+	return organization, nil
+}
+
+func (m *Repository) AddOrganization(ctx context.Context, tx persistence.TransactionHandler, organization *model.Organization) (*model.Organization, error) {
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return nil, fmt.Errorf("extract context executor: %v", err)
+	}
+
+	entry := &mysqlmodel.Organization{
+		Name:      organization.Name,
+		CreatedBy: organization.CreatedBy,
+	}
+
+	if err = entry.Insert(ctx, ctxExec, boil.Infer()); err != nil {
+		return nil, fmt.Errorf("insert organization: %v", err)
+	}
+
+	organization, err = m.GetOrganizationById(ctx, tx, entry.ID)
+	if err != nil {
+		return nil, fmt.Errorf("get organization by id: %v", err)
+	}
+
+	return organization, nil
+}
+
+func (m *Repository) RestoreOrganization(
+	ctx context.Context,
+	tx persistence.TransactionHandler,
+	id int,
+) error {
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return fmt.Errorf("get ctx exec: %v", err)
+	}
+
+	entry := &mysqlmodel.Organization{ID: id, IsActive: true}
+	if _, err = entry.Update(ctx, ctxExec, boil.Whitelist("is_active")); err != nil {
```

Comment:
use [mysqlmodel](https://github.com/dembygenesis/theoverwatchtools.cli/blob/33dd61bccf6819341626cb14f1736f6bf80589f2/internal/persistence/persistors/mysqlstore/store_category.go#L60)

Reply:
This has now been changed to boil.Whitelist(mysqlmodel.OrganizationColumns.IsActive))